### PR TITLE
Ethernet stats monitor

### PR DESCRIPTION
**Pull request by pfeerick**
I've done this as a separate branch/PR so you don't have to cherrypick commits. I was knocking around in the sysfs space and came across ethernet stats, and though it might be useful. If it's now that interesting, it's in my scripts repo anyway.

Displays information from the ethernet statistics sys path.
Formatting and display code based on rock64_health.sh origins.

### CODE REVIEW

File: package/root/usr/local/sbin/rock64_eth0_stats.sh [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        echo "This script requires root."
+        exit 1
+fi
+
+print() {
+        printf "%-20s: %s %s\n" "$1" "$2 $3" "$4"
+}
+
+statprint() {
+	local STAT_PATH=/sys/bus/platform/drivers/rk_gmac-dwmac/ff540000.eth/net/eth0/statistics
```
Comment (ayufan) [inline package/root/usr/local/sbin/rock64_eth0_stats.sh]: Should this be a global variable?

File: package/root/usr/local/sbin/rock64_eth0_stats.sh [hunk 2]
```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+set -e
+
+if [ "$(id -u)" -ne "0" ]; then
```
Comment (ayufan) [inline package/root/usr/local/sbin/rock64_eth0_stats.sh]: What of here requires root?

File: package/root/usr/local/sbin/rock64_eth0_stats.sh [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+}
+
+usage() {
+	echo "Usage: $0 [-w] [-h]"
+}
+
+WATCH=""
+for i in "$@"; do
+	case $i in
+	-w)
```
Comment (ayufan) [inline package/root/usr/local/sbin/rock64_eth0_stats.sh]: `getopt`?

Comment (ayufan) [inline package/root/usr/local/sbin/rock64_eth0_stats.sh]: http://wiki.bash-hackers.org/howto/getopts_tutorial

File: package/root/usr/local/sbin/rock64_eth0_stats.sh (hunk 2 above)
Comment (pfeerick) [inline package/root/usr/local/sbin/rock64_eth0_stats.sh]: In this case, nothing, it was a leftover of the original health script. Permissions in this case allow 'everyone' to read.

File: package/root/usr/local/sbin/rock64_eth0_stats.sh (hunk 1 above)
Comment (pfeerick) [inline package/root/usr/local/sbin/rock64_eth0_stats.sh]: Isn't needed, but I suppose it makes sense.

File: package/root/usr/local/sbin/rock64_eth0_stats.sh (hunk 3 above)
Comment (pfeerick) [inline package/root/usr/local/sbin/rock64_eth0_stats.sh]: New script, yeah, I'll switch to `getopt` for this one.

**Pull request merged**